### ISSUE TITLE
Loading `JSON` / `Metrics Explorer` with 1600+ metrics takes 750+ms

### ISSUE THREAD

**Issue by spmcbride1201** [DecipherNow/gm-fabric-dashboard#111]
```
[Violation] 'click' handler took 499ms
[Violation] Forced reflow while executing JavaScript took 253ms
```

**Comment by spmcbride1201** [comment]
This might have to do with the JSON component itself. Perhaps we might want to replace this with an alternate component better suited to flat arrays of keys.

**Comment by spmcbride1201** [comment]
This was closed in `0.1.0`
